user-manual/administer/settings.rst

```diff
@@ -1871,3 +1872,132 @@ setting.
+
+     .. code-block:: none
+
+        This website uses cookies to enhance your ability to browse and load
+        content. This website uses cookies to enhance your ability to browse
+        and load content. More Info: http://10.10.10.10/privacy
+
+Administrators can edit the text of the banner message, and choose whether or
+not to display the banner when a user accesses your site for the first time.
+Here's an example of how the notification banner will look:
+
+.. image:: images/gdpr-banner-post-set.*
+   :align: center
+   :width: 80%
+   :alt: GDPR banner first access
+
+* Update the URL in the message to the base URL for your site (see :ref:`Site
+  information <site-information>`)
+* If desired, you can use the current custom link syntax formatting to style the
+  hyperlink -  see :ref:`formatting` for more information
+
+Edit the Privacy Policy
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Administrators should also review the wording of the new default Privacy Policy
+static page. This can be found in |gears| *Admin > Static Pages >Privacy
+Policy*, or it can be accessed via the Quick links menu in the AtoM header.
+Click on ``edit`` at the bottom of the view page, make your desired edits, and
+select "save" to save your changes or ``cancel`` to return to the default text.
+
+Upgrades to 2.4.1
+-----------------
+
+After first upgrading the cookie notification banner will appear as follows:
+
+.. image:: images/gdpr-banner-pre-task.*
+   :align: center
+   :width: 80%
+   :alt: GDPR banner first access
+
+* Additionally, if no further action is taken, the new default Privacy policy
+  static page will not exist
+* If you don't intend to use the banner or need the Privacy policy static page,
+  you can disable the notification banner via |gears| *Admin > Settings >
+  Privacy policy*
+* To create the Privacy policy static page and update the banner message, a
+  system administrator should run the following task from the root AtoM
+  installation directory:
+
+.. code-block:: bash
+
+   php symfony tools:runlib/task/tools/addGdprSettings.php
+
+Custom theme updates
+--------------------
+
+* If you have a custom theme and the file ``scaffolding.less`` has been
+  customized, this file will need to be updated. Changes are identified
+  `here <https://goo.gl/d6HVVf>`_ - see the changes to ``scaffolding.less``
+* If a theme has been customized, but the file ``scaffolding.less`` is being
+  referenced from the arDominion theme, then there are no modifications
+  required.
+
+  * This can be checked in your theme's
+    ``plugins/<themefolder>/css/main.less`` file.  Look for:
+    ``@import "../../arDominionPlugin/css/less/scaffolding.less";`` -
+    if this line is present, no modifications should be required
+
+* Check if ``_header.php`` has been overridden in the custom theme. If so, the
+  change highlighted in this `issue <https://github.com/artefactual/atom/commit/c65e84e809a5760c9814f8117a291bdb9a7491da#diff-e3a653026878cbc4745a5526934888d7will>`_
```

Comment:
Two small things here - one formatting typo, and one weird thing. 

First, I tested the link that was here (and on the wiki), and for whatever reason, it wasn't properly highlighting the relevant line! The correct link to be used should be: 

* https://github.com/artefactual/atom/commit/c65e84e809a5760c9814f8117a291bdb9a7491da#diff-e3a653026878cbc4745a5526934888d7R3

I've fixed it on the wiki now. 

Second, there's a little typo here - you have accidentally include the word "will" in the URL!